packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/normal/import_assetic/import_with_parameter.yaml

```diff
@@ -0,0 +1,12 @@
+imports:
+    - { resource: '%kernel.project_dir%/somefile.yaml' }
```

Comment:
reference for this usecase :https://symfony.com/doc/6.1/service_container/import.html

Reply:
Could the yaml has comment with `@see` ? , something like this:

```
# @see https://symfony.com/doc/6.1/service_container/import.html
```

if can, please ad so that wil ease to find out why in the future.

Reply:
Added. What can I do about the complexity report. Not sure how to keep it readable.

Reply:
You can move this code to separate private method:

```php
       if (is_bool($value)) {
            return BuilderHelpers::normalizeValue($value);
        }
        if (in_array($value, ['annotations', 'directory', 'glob'], true)) {
            return BuilderHelpers::normalizeValue($value);
        }
```

eg: 

```php
private function processNormalizedValue(mixed $value): ?Expr
{
        if (is_bool($value)) {
            return BuilderHelpers::normalizeValue($value);
        }
        
        if (in_array($value, ['annotations', 'directory', 'glob'], true)) {
            return BuilderHelpers::normalizeValue($value);
        }

       return null;
}
```

so, you now reduced 1 if:

```php
$normalizedValue = $this->processNormalizedValue($value);
if ($normalizedValue instanceof Expr) {
    return $normalizedValue;
}

// the rest of other code here
```

```php
$normalizedValue = $htis->processNormalizedValue
```

Reply:
Thx that did do the job :)